tests/server/config/config_test.go

```diff
@@ -491,11 +491,11 @@ func (suite *configTestSuite) assertTTLConfigItemEqaul(
 		}
 		return false
 	}
-	suite.True(checkfunc(cluster.GetLeaderServer().GetServer().GetPersistOptions()))
+	suite.True(checkFunc(cluster.GetLeaderServer().GetServer().GetPersistOptions()))
 	if cluster.GetSchedulingPrimaryServer() != nil {
 		// wait for the scheduling primary server to be synced
```

Comment:
The check here does not include ttl expired cases? Because there was no waiting.